hpx/runtime/actions/continuation.hpp

```diff
@@ -103,7 +104,7 @@ namespace hpx
 ///////////////////////////////////////////////////////////////////////////////
 namespace hpx { namespace actions
 {
-    class HPX_EXPORT continuation;
+    class HPX_API_EXPORT continuation;
```

Comment:
Even if `HPX_EXPORT` and HPX_API_EXPORT` currently expand to the same things, I'd prefer leaving `HPX_EXPORT` for type exports and `HPX_API_EXPORT` for function exports.